### PR TITLE
Create REST API

### PR DESCRIPTION
**Pull request by ArielxX**
Create REST API using fastAPI and integrate it with docker by using gunicorn

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,9 +8,12 @@ click==8.1.6
 contourpy==1.1.0
 cycler==0.11.0
 docopt==0.6.2
+dvc==3.15.3
```
Comment (aniervs) [inline requirements.txt:11]: I see you added DVC as a package in the project. When I installed DVC as a python package on my machine, I had some problems, which were solved when installing it through apt.

Comment (aniervs) [reply requirements.txt:11]: Maybe it's a good idea to not have it as package.

**Review by aniervs — CHANGES_REQUESTED** [review]
Besides the other comments I left in the other files, I have nothing left to say.

It's worth noticing that the cropping should be also here.
What do you think works better?
1. Waiting for the cropping to be on the main and then merging it locally before the PR.
2. Accept your PR, and then I change the API according to the new feature of cropping.

File: requirements.txt (hunk 1 above)
Comment (ArielxX) [reply requirements.txt]: done

**Review by aniervs — APPROVED** [review]

**Pull request merged**